src/core/core.tooltip.js

```diff
@@ -701,10 +705,8 @@ module.exports = function(Chart) {
 						// Inner square
 						ctx.fillStyle = mergeOpacity(vm.labelColors[i].backgroundColor, opacity);
```

Comment:
you wouldn't need this line since you end up overriding it later

Reply:
Thank you for reviewing.
I revert it and fixed up. :sushi: